mint/run/core/aws-sdk-go/quick-tests.go

```diff
@@ -609,6 +609,114 @@ func testObjectTagging(s3Client *s3.S3) {
 	}
 }
 
+// Tests bucket re-create errors.
+func testCreateBucketError(s3Client *s3.S3) {
+	// initialize logging params
+	startTime := time.Now()
+	function := "testMakeBucketError"
+	bucketName := randString(60, rand.NewSource(time.Now().UnixNano()), "aws-sdk-go-test-")
+	args := map[string]interface{}{
+		"bucketName": bucketName,
+	}
+	_, err := s3Client.CreateBucket(&s3.CreateBucketInput{
+		Bucket: aws.String(bucketName),
+	})
+	if err != nil {
+		failureLog(function, args, startTime, "", "AWS SDK Go CreateBucket Failed", err).Fatal()
+		return
+	}
+	defer cleanup(s3Client, bucketName, "", function, args, startTime, true)
+
+	_, errCreating := s3Client.CreateBucket(&s3.CreateBucketInput{
+		Bucket: aws.String(bucketName),
+	})
+	if errCreating == nil {
+		failureLog(function, args, startTime, "", "AWS SDK Go CreateBucket Should Return Error for Existing bucket", err).Fatal()
+		return
+	}
+	// Verify valid error response from server.
+	if errCreating.(s3.RequestFailure).Code() != "BucketAlreadyExists" &&
```

Comment:
AWS S3 returns success if the region is `us-east-1`

Reply:
Yes that's why we need to use a different region in this test.

Reply:
@kannappanr @harshavardhana any suggested region to use in this test ?

Reply:
any region which is non us-east-1 will work @Nesty92

Reply:
@kannappanr @harshavardhana play.minio.io is restricted yo us-east-1?. I get an error when I try to change the region for this test 
```sh
"error":"InvalidRegion: Region does not match.\n\tstatus code: 400
```

Reply:
Most probably, let's fix that to allow all regions.

Reply:
You should be able to change this now.